public/mentor/index.html

```diff
@@ -0,0 +1,340 @@
+      <header class="p-6">
+        <div
+          class="container flex justify-between items-center mx-auto px-8 md:px-14 lg:px-24 w-full"
+        >
+          <div class="logo-area">
+            <img
+              src="../_assets/images/logo.svg"
+              alt="logo"
+              class="h-10 ml-30"
+            />
+          </div>
+          <div class="hidden md:flex items-center">
+            <a
+              href="./about/"
+              class="block text-primary-dark lg:inline-block header-tag"
+            >
+              關於我們
+            </a>
+            <a
+              href="./co-study/"
+              class="block text-primary-dark lg:inline-block header-tag"
+            >
+              共學
+            </a>
+            <a
+              href="./co-create/"
+              class="block text-primary-dark lg:inline-block header-tag"
+            >
+              共創
+            </a>
+            <a
+              href="./community/"
+              class="block text-primary-dark lg:inline-block header-tag"
+            >
+              社群
+            </a>
+            <a
+              href="#responsive-header"
+              class="block text-primary-dark lg:inline-block header-tag"
+            >
+              加入我們
+            </a>
+            <a
+              href="#responsive-header"
+              class="block text-primary-dark lg:inline-block header-tag"
+            >
+              登入會員
+            </a>
+          </div>
+          <div class="md:hidden">
+            <img
+              src="../_assets/images/backgrounds/hamburger-menu.svg"
+              alt=""
+            />
+          </div>
+        </div>
+      </header>
+
+      <!--Mentor Intro-->
+      <div
+        class="container mt-28 flex justify-between items-center mx-auto px-8 md:px-14 lg:px-24 w-full"
+      >
+        <section class="w-full">
+          <div class="flex flex-row justify-start font-nenu">
+            <span class="text-h2 font-medium">第 </span>
+            <span
+              class="text-h2 font-medium border-b-4 border-secondary-default pl-2"
+            >
+              10 期</span
+            >
+            <span class="text-h2 font-medium">導師</span>
+          </div>
+          <div class="space-y-12 my-14 font-nenu">
+            <!-- class="w-full border-2 border-darkgray md:px-4 md:py-7 py-0 px-0 lg:space-x-12 md:flex justify-center lg:justify-start flex-wrap lg:flex-nowrap" -->
+            <div
+              class="border-2 border-border-darkgray px-2 py-2 md:px-4 md:py-7 md:flex bg-white"
+            >
+              <img
+                src="../_assets/images/mentors/example.svg"
+                alt="example"
+                width="320"
+                height="320"
+              />
+              <div
+                class="flex flex-wrap justify-start text-left lg:text-left lg:block"
+              >
+                <div>
+                  <span class="intro-decoration">——</span>
```

Comment:
There're many ways to draw this blue segment but using texts would be the bad one.

- Draw with SVG.
- Draw with CSS.

Choose one.